File: geek-university/algoritmo-e-logica-de-programacao/c/exercicio_30.c
#include <stdio.h>

int main() {
    int numero;

    printf("Informe um numero entre 1 e 10: ");
    scanf("%d", &numero);

    while (numero < 1 || numero > 10) {
        printf("Informe um numero entre 1 e 10: ");
        scanf("%d", &numero);
    }

    for (int i = 1; i <= 10; i++) {
        printf("%d X %d = %d\n", numero, i, (numero * i));
    }
}

File: geek-university/algoritmo-e-logica-de-programacao/java/exercicio_30.java
import java.util.Scanner;

public class exercicio_30 {

    public static void main(String[] args) {
        int numero;
        Scanner teclado = new Scanner(System.in);

        System.out.print("Informe um numero entre 1 e 10: ");
        numero = teclado.nextInt();

        while (numero < 1 || numero > 10) {
            System.out.print("Informe um numero entre 1 e 10: ");
            numero = teclado.nextInt();
        }

        for (int i = 1; i <= 10; i++) {
            System.out.printf("%d X %d = %d\n", numero, i, (numero * i));
        }

        teclado.close();
    }

}
